e2e/apprunner/copilot/back-end/manifest.yml

```diff
@@ -0,0 +1,61 @@
+# The manifest for the "back-end" service.
```

Comment:
Why do we add `back-end` and `test` env manifest?

Reply:
Removed!

Reply:
doesn't seem to be removed tho.

Reply:
removed

Reply:
Added `e2e-apprunner-MyTestSecret` (secrets manager secret) to backend service.